core/services/chainlink/config_test.go

```diff
@@ -1331,7 +1331,9 @@ func TestConfig_Validate(t *testing.T) {
 		- 1.ChainID: invalid value (1): duplicate - must be unique
 		- 0.Nodes.1.Name: invalid value (foo): duplicate - must be unique
 		- 3.Nodes.4.WSURL: invalid value (ws://dupe.com): duplicate - must be unique
-		- 0: 3 errors:
+		- 0: 5 errors:
+			- logbroadcaster cannot be enabled unless all nodes have WSURL provided
```

Comment:
Is this true that _all_ nodes must have WSURL? The error message in toml/config.go says: `must have at least one`